src/components/Project/FeedbackModule.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import Section from '../General/Section';
+import LongCard from '../General/LongCard';
+import styled from 'styled-components';
+import {breakpoint} from '@aragon/ui';
+const medium = css => breakpoint('medium', css);
+const large = css => breakpoint('large', css);
+import speech from './assets/speech.svg';
+
+const FeedbackModule = () => (
+  <StyledSection>
+    <Container>
+      <Feedback>
+        <ImageContainer>
+          <img src={speech} />
+        </ImageContainer>
+        <div>
+          <h1>We value the feedback & ideas from the community</h1>
+          <span>
+            <p>
+              Please let us know if there something important for you that is
+              not included in this roadmap and we’ll take it into consideration
+              when planning our quaterly efforts.
+            </p>
+            <p>
+              <a
+                className="action-button"
+                href="https://forum.aragon.org"
+                target="_blank">
+                <span>
+                  Join the Aragon forum{' '}
+                  <img src={require(`../General/assets/arrow.svg`)} />
+                </span>
+              </a>
+            </p>
+          </span>
+        </div>
+      </Feedback>
+    </Container>
+  </StyledSection>
+);
+
+const StyledSection = styled(Section)`
+    padding: 100px 15px;
+`
+const Container = styled.div`
+  display: grid;
```

Comment:
❤️ 
May I suggest you re-structure your code to add the `medium`/`large` overrides after you've set all the base rules? It'll help you visually understand the breakpoint right away:

```JavaScript
const Container = styled.div`
  display: grid;
  grid-template-columns: 100%;
  grid-template-rows: auto;
  grid-column-gap: 30px;
  grid-row-gap: 30px;

  ${medium(
    'grid-template-columns: calc(50% - 15px) calc(50% - 15px); grid-template-rows: auto'
  )};
`;
```